src/BitcoinLib/RPC/Connector/IRpcConnector.cs

```diff
@@ -1,12 +1,13 @@
 ﻿// Copyright (c) 2014 - 2016 George Kimionis
 // See the accompanying file LICENSE for the Software License Aggrement
 
+using System.Threading.Tasks;
 using BitcoinLib.RPC.Specifications;
 
 namespace BitcoinLib.RPC.Connector
 {
     public interface IRpcConnector
     {
-        T MakeRequest<T>(RpcMethods method, params object[] parameters);
+        Task<T> MakeRequestAsync<T>(RpcMethods method, params object[] parameters);
```

Comment:
Is it *required* by .NET Core to have an async version of that call? Wouldn't it be easier if it's done on a separate pull request?